cmd/openshift-install/analyze.go

```diff
@@ -27,6 +27,8 @@ func newAnalyzeCmd(ctx context.Context) *cobra.Command {
 This command helps users to analyze the reasons for an installation that failed while bootstrapping.`,
 		Args: cobra.ExactArgs(0),
 		Run: func(_ *cobra.Command, _ []string) {
+			_, _ = handleInterrupt(ctx, exitOnInterrupt)
```

Comment:
@vincepri does this seem like the right pattern? I'm plumbing in the interrupt handler & context to every command. If that command doesn't accept a context, the context is now available here for future work. 

I think we need to setup a handler for each command in order to break out of the signal handler trap, so users don't need to CTRL+C twice. Some of these commands run very quickly so a user interrupt is unlikely and they will likely not need a context; but I figure it is better to add the handlers to each command and a new context won't hurt.